app/presenters/document/record_presenter.rb

```diff
@@ -6,11 +6,11 @@ def edm_resource_url
       @edm_resource_url ||= render_document_show_field_value('aggregations.edmIsShownBy')
     end
 
-    def media_web_resources(options)
+    def media_web_resources(options = {})
```

Comment:
Assignment Branch Condition size for media_web_resources is too high. [31.4/15]<br>Method has too many lines. [14/10]